doc/testing.md

```diff
@@ -66,3 +64,31 @@ Also test watcher can be launched. It will re-run the entire test suite when any
 ```
 
 Check [component tests doc](./component-tests-overview.md) for more.
+
+## Unit tests only
+
+To run unit tests:
+
+```
+   make unit-test
+```
+
+Also test watcher can be launched. It will re-run the entire test suite when any file is modified
+
+```
+   make unit-test WATCH=true
+```
+
+## Integration tests only
+
+To run integration tests:
+
+```
+   make integration-test
```

Comment:
I think it's `make test-integration`
or perhaps both work? @ilmotta 🤔 ?

Reply:
ah I picked this up from the PR.
I'll check the Makefile.

Reply:
updated, thanks for spotting this @J-Son89